src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -96,7 +96,7 @@ private void testContent() throws Exception {
 
         // Verify with retry after upload picture due to inconsistency of .click in detecting page load
         profilePage.verifyStatusWithRetry(Const.StatusMessages.STUDENT_PROFILE_PICTURE_SAVED, 10);
-        profilePage.verifyHtmlPart(By.className("profile-pic-edit-toolbar"), "/studentProfilePictureModalEdit.html");
+        profilePage.verifyHtml("/studentProfilePageFilled.html");
```

Comment:
The reason the original issue was raised was so that we can have test cases to cover the entire content of the page instead of just selected parts of it. It was hindered because no viable way was available to place-hold the blob-key